content/docs/conditional-rendering.md

```diff
@@ -36,43 +34,42 @@ function Greeting(props) {
 }
 
 ReactDOM.render(
-  // Try changing to isLoggedIn={true}:
+  // Kodu değiştirerek deneyin: isLoggedIn={true}:
   <Greeting isLoggedIn={false} />,
   document.getElementById('root')
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ZpVxNq?editors=0011)
+[**CodePen üzerinde deneyin**](https://codepen.io/gaearon/pen/ZpVxNq?editors=0011)
```

Comment:
We decided to translate this as `CodePen'de Deneyin`. So it is better if you change it everywhere. Thanks.